client/src/main/scala/drt/client/components/FeatureGuideComponent.scala

```diff
@@ -0,0 +1,107 @@
+package drt.client.components
+
+import drt.client.SPAMain
+import drt.client.components.styles.{DrtTheme, WithScalaCssImplicits}
+import drt.client.modules.GoogleEventTracker
+import drt.client.services.DrtApi
+import io.kinoplan.scalajs.react.material.ui.core.MuiButton._
+import io.kinoplan.scalajs.react.material.ui.core._
+import io.kinoplan.scalajs.react.material.ui.core.system.{SxProps, ThemeProvider}
+import japgolly.scalajs.react.component.Scala.Component
+import japgolly.scalajs.react.vdom.html_<^.{<, ^, _}
+import japgolly.scalajs.react.{BackendScope, Callback, CtorType, ReactEvent, ScalaComponent}
+import upickle.default.write
+
+import scala.language.postfixOps
+import io.kinoplan.scalajs.react.material.ui.core.{MuiDialog, MuiDialogContent, MuiDialogTitle}
+import uk.gov.homeoffice.drt.training.FeatureGuide
+
+
+object FeatureGuideModalComponent extends WithScalaCssImplicits {
+
+  case class State(currentStep: Double)
+
+  case class Props(showDialog: Boolean, closeDialog: ReactEvent => Callback, trainingDataTemplates: Seq[FeatureGuide])
+
+  class Backend($: BackendScope[Props, State]) {
+
+    def handleOnPlayVideo(filename: String)(e: ReactEvent): Callback = {
+      {
```

Comment:
I think it would make more sense to send the ID of the feature rather than the filename
The function should be called `recordFeatureGuideView` - it doesn't handle the playing of the video

Reply:
Can remove both sets of curly braces here